iv.yaml

```diff
@@ -12,6 +12,8 @@ test:
   - types/models/comanda_test.go
   - types/models/optimizador_test.go
 
+configuracion: .env
```

Comment:
Vamos a ver, si se trata de que no lo añadáis en el repo no es para que lo pongas aquí... 
Lo que tienes que poner es el fichero donde has programado la configuración.

Reply:
Cambiado y creada la abstracción de la lectura de la configuración